tools/ci_connector_ops/ci_connector_ops/pipelines/gradle.py

```diff
@@ -69,23 +41,6 @@ def docker_service_name(self) -> str:
     def connector_java_build_cache(self) -> CacheVolume:
         return self.context.dagger_client.cache_volume("connector_java_build_cache")
 
-    def get_related_connectors(self) -> List[Connector]:
-        """Retrieve the list of related connectors.
-        This is used to include source code of non strict-encrypt connectors when running build for a strict-encrypt connector.
-
-        Returns:
-            List[Connector]: The list of related connectors.
-        """
-        if self.context.connector.technical_name.endswith("-strict-encrypt"):
-            return [Connector(self.context.connector.technical_name.replace("-strict-encrypt", ""))]
-        if self.context.connector.technical_name == "source-file-secure":
-            return [Connector("source-file")]
-        if self.context.connector.technical_name == "destination-bigquery-denormalized":
-            return [Connector("destination-bigquery")]
-        if self.context.connector.technical_name == "destination-dev-null":
-            return [Connector("destination-e2e-test")]
-        return []
```

Comment:
Can you confirm (do a test) that changing a 'regular' connector also knows to publish the strict encrypt version?

Reply:
To verify that I'm getting what's going on here. What you're saying @evantahler should happen since a build.gradle in a strict-encrypt flavor has a dependency for its non strict-encrypt.
e.g. in source-postgres-strict-encrypt you got 
```implementation project(':airbyte-integrations:connectors:source-postgres')```